todo_app/app/views/todos/show.html.erb

```diff
@@ -0,0 +1,5 @@
+
+<h2><%= link_to "Update", edit_todo_path %>
+<%= form_for @todo, method: :delete do |f| %>
+  <%= f.submit "Remove", class: "delete" %>
+<% end %>
```

Comment:
Would recommend also displaying the individual todo on this show page. Good job on adding the update and delete capabilities. 👍